src/main/java/sql/dao/mysql/EmployeeDAO.java

```diff
@@ -104,4 +103,19 @@ public void delete(int id) {
         deleteWithTryCatch(sql, id);
 
     }
+
+    @Override
+    public void deletePhone(int id) {
+
+    }
+
+    @Override
+    public void updatePhone(Employee o, int id) {
+
+    }
+
+    @Override
+    public void savePhone(Employee o) {
+
+    }
```

Comment:
The method was inherited by the interface. I added this method so it will work as intended for MyBatis. In this case, the way I implemented the MySQL, makes this method unnecessary.